gc/base/standard/Scavenger.cpp

```diff
@@ -5738,6 +5740,8 @@ void MM_Scavenger::postConcurrentUpdateStatsAndReport(MM_EnvironmentBase *env, M
 	OMRPORT_ACCESS_FROM_OMRPORT(env->getPortLibrary());
 
 	_cycleTimes.incrementEnd = omrtime_hires_clock();
+	stats->_incrementStartTime = _cycleTimes.incrementStart;
```

Comment:
This can be done in `preConcurrentInitializeStatsAndReport` right after setting `_cycleTimes.incrementStart `